### DISCUSSION TITLE
[Feature] Option to compress uploading images (Android\iOS)

### DISCUSSION THREAD

**Discussion by ratso** [immich-app/immich#4559] [Feature Request]
### The feature

With Google Photos you can upload images in its original size or compressed to 16Mp.  It's handy when you shoot alot and no need to have photos in its original resolution.
It will be great to have this option too. For example, an option to enable photo compression and define maximum MP size or dimensions of image. Conversion should be performed on user device automatically.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of  #1202

**Reply by ratso** [reply]
Why not? Option could be disabled by default. It should be done on iOS/android app.
